modules/configuration/OverallStateCollectorConfig.py

```diff
@@ -0,0 +1,11 @@
+import os
+import logging
+
+logger = logging.getLogger('esxi')
+
+
+class OverallStateCollectorConfig:
+
+    def __init__(self):
+        self.vcenter_username = os.environ['VCENTER_USER']
+        self.vcenter_password = os.environ['VCENTER_PASSWORD']

```

Comment:
Can we integrate a master password here? 
If VCENTER_MPW is set, then use a generated master_password password, if not, use VCENTER_PASSWORD